README.md

```diff
@@ -1 +1,2 @@
-# Leaderboard
\ No newline at end of file
+# Leaderboard
+
```

Comment:
- [ ] Please kindly make your `README` file more professional, you can use this [templante](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse 🙃 🙃

Reply:
Good job following the advice of the previous reviewer